03-Purpose-and-Principles/Documentation-Principles.md

```diff
@@ -0,0 +1,46 @@
+# Documentation Principles
+
+These principles are not set in stone -- if you'd like clarification, qualification, addition, removal, or changes to these principles, open an issue to discuss.
+
+## Documentation Scope
+
+- There are many types of applications that can be built with PureScript, each of which should have documentation. The group of people who write each type of application is best suited for managing its documentation.
+- All types of PureScript applications have pre-requisite knowledge, and the set of this knowledge which is common across all PureScript applications should be managed by the group of PureScript groups, an entity which we will name the PureScript organization.
```

Comment:
I'm really not sure what this is trying to say. Could you clarify?

Reply:
Perhaps also addressed by https://github.com/chexxor/purescript-documentation-discussion/pull/69/commits/ef38c4a06726eb5b95ebcfb2fb96044587bcf8b2?